api/AmbientLightSensorReading.json

```diff
@@ -0,0 +1,106 @@
+{
+  "api": {
+    "AmbientLightSensorReading": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensorReading",
+        "support": {
+          "webview_android": {
+            "version_added": "54"
+          },
+          "chrome": {
+            "version_added": "54"
+          },
+          "chrome_android": {
+            "version_added": "54"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AmbientLightSensorReading": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensorReading/AmbientLightSensorReading",
```

Comment:
I've been adding a description to constructor entries - ie 
`"description": "<code>AmbientLightSensorReading()</code> constructor",`
I'm not sure how important that is, but thought I'd mention.

Reply:
:heavy_check_mark: